make test is failing

When I run make test I'm getting this error
```bash
$ make test
go get github.com/mattn/goveralls
go get github.com/modocache/gover
go get github.com/Masterminds/glide
go get github.com/sgotti/glide-vc
go get github.com/golang/lint/golint
./script/check-vendor.sh
Checking for nested vendor dirs
OK
Checking if vendor was cleaned using glide-vc.
OK
./script/check-gofmt.sh
gofmt OK
go vet ./pkg/... ./cmd/... .
pkg/loader/compose/compose_test.go:67: arg expected for printf verb %s of wrong type: github.com/kubernetes-incubator/kompose/pkg/kobject.Ports
exit status 1
Makefile:75: recipe for target 'vet' failed
make: *** [vet] Error 1
```

--- a/pkg/loader/compose/compose_test.go
+++ b/pkg/loader/compose/compose_test.go
@@ -64,7 +64,7 @@ func TestLoadV3Ports(t *testing.T) {
 	}
 
 	if output[0] != expected {
-		t.Errorf("Expected %s, got %s", expected, output[0])
+		t.Errorf("Expected %v, got %v", expected, output[0])
 	}
 }
 

